ccl/kvccl/kvtenantccl: TestTenantUpgradeInterlock failed

ccl/kvccl/kvtenantccl.TestTenantUpgradeInterlock [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138148?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138148?buildTab=artifacts#/) on master @ [1cf75095871a9df935c036c674783dca73a02134](https://github.com/cockroachdb/cockroach/commits/1cf75095871a9df935c036c674783dca73a02134):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 360690 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantUpgradeInterlock
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/416c130b9e53a570f2915b8546e7d23c/logTestTenantUpgradeInterlock3429012815
    test_log_scope.go:79: use -show-logs to present logs inline
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after write to settings table"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_migration
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after migration"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after version bump RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_migration
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after migration"
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/multi-tenant
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantUpgradeInterlock.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-25640

ccl/kvccl/kvtenantccl.TestTenantUpgradeInterlock [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138888?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138888?buildTab=artifacts#/) on master @ [65698efb7bc7e63d23ea43e43d3a74627608e657](https://github.com/cockroachdb/cockroach/commits/65698efb7bc7e63d23ea43e43d3a74627608e657):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 400109 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantUpgradeInterlock
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/416c130b9e53a570f2915b8546e7d23c/logTestTenantUpgradeInterlock2483010096
    test_log_scope.go:79: use -show-logs to present logs inline
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after version bump RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after write to settings table"
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantUpgradeInterlock.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/kvccl/kvtenantccl.TestTenantUpgradeInterlock [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138888?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138888?buildTab=artifacts#/) on master @ [65698efb7bc7e63d23ea43e43d3a74627608e657](https://github.com/cockroachdb/cockroach/commits/65698efb7bc7e63d23ea43e43d3a74627608e657):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 369906 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantUpgradeInterlock
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/416c130b9e53a570f2915b8546e7d23c/logTestTenantUpgradeInterlock135405345
    test_log_scope.go:79: use -show-logs to present logs inline
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_migration
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after migration"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after version bump RPC"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after version bump RPC"
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantUpgradeInterlock.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/kvccl/kvtenantccl.TestTenantUpgradeInterlock [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138888?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9138888?buildTab=artifacts#/) on master @ [65698efb7bc7e63d23ea43e43d3a74627608e657](https://github.com/cockroachdb/cockroach/commits/65698efb7bc7e63d23ea43e43d3a74627608e657):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 396909 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantUpgradeInterlock
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/416c130b9e53a570f2915b8546e7d23c/logTestTenantUpgradeInterlock2216206795
    test_log_scope.go:79: use -show-logs to present logs inline
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after write to settings table"
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantUpgradeInterlock.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/kvccl/kvtenantccl.TestTenantUpgradeInterlock [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9140483?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9140483?buildTab=artifacts#/) on master @ [9acc75317aebfdfe69fb097a8a28e0040c7a67fd](https://github.com/cockroachdb/cockroach/commits/9acc75317aebfdfe69fb097a8a28e0040c7a67fd):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 325608 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantUpgradeInterlock
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/416c130b9e53a570f2915b8546e7d23c/logTestTenantUpgradeInterlock3328204600
    test_log_scope.go:79: use -show-logs to present logs inline
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after write to settings table"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_migration
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after migration"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after version bump RPC"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after second check of instances"
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantUpgradeInterlock.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/kvccl/kvtenantccl.TestTenantUpgradeInterlock [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9140483?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9140483?buildTab=artifacts#/) on master @ [9acc75317aebfdfe69fb097a8a28e0040c7a67fd](https://github.com/cockroachdb/cockroach/commits/9acc75317aebfdfe69fb097a8a28e0040c7a67fd):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 380974 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantUpgradeInterlock
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/416c130b9e53a570f2915b8546e7d23c/logTestTenantUpgradeInterlock2873138289
    test_log_scope.go:79: use -show-logs to present logs inline
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after version bump RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after write to settings table"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_migration
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after migration"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after version bump RPC"
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantUpgradeInterlock.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/kvccl/kvtenantccl.TestTenantUpgradeInterlock [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9150232?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9150232?buildTab=artifacts#/) on master @ [dd7766e51bc77b76d424dca463e0dc6e58e21f87](https://github.com/cockroachdb/cockroach/commits/dd7766e51bc77b76d424dca463e0dc6e58e21f87):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 354904 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantUpgradeInterlock
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/416c130b9e53a570f2915b8546e7d23c/logTestTenantUpgradeInterlock140123678
    test_log_scope.go:79: use -show-logs to present logs inline
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence RPC"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_fence_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after fence write to settings table"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_first_check_for_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after first check for instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_version_bump_RPC
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after version bump RPC"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_second_check_of_instances
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after second check of instances"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_migration
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after migration"
=== RUN   TestTenantUpgradeInterlock/current_binary_version_pause_after_write_to_settings_table
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "current binary version", configuration: "pause after write to settings table"
=== RUN   TestTenantUpgradeInterlock/lagging_binary_version_pause_after_migration
    tenant_upgrade_test.go:525: upgrade interlock test: running variant "lagging binary version", configuration: "pause after migration"
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantUpgradeInterlock.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/kvccl/kvtenantccl/tenant_upgrade_test.go
+++ b/pkg/ccl/kvccl/kvtenantccl/tenant_upgrade_test.go
@@ -403,7 +403,6 @@ func TestTenantUpgradeFailure(t *testing.T) {
 // is too low.
 func TestTenantUpgradeInterlock(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	skip.WithIssue(t, 98987)
 	// Times out under stress race
 	skip.UnderStressRace(t)
 	// Test takes 30s to run

